pkg/users/account-details.js

```diff
@@ -230,119 +231,112 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
-                                        <AccountRoles account={account} groups={groups}
-                                            currently_logged_in={details.logged.currently} />
-                                    </div>
-                                </td>
-                            </tr>
+        <Page id="account"
+              breadcrumb={
+                  <Breadcrumb>
+                      <BreadcrumbItem onClick={() => cockpit.location.go("/")} className="pf-c-breadcrumb__item" to="#">{_("Accounts")}</BreadcrumbItem>
+                      <BreadcrumbItem isActive>{title_name}</BreadcrumbItem>
+                  </Breadcrumb>}>
+            <PageSection>
+                <Gallery hasGutter>
+                    <Card className="account-details" id="account-details">
+                        <CardHeader>
+                            { superuser.allowed &&
+                            <CardActions>
+                                <Button variant="secondary" onClick={() => logout_account()} id="account-logout"
+                                  isDisabled={!details.logged.currently || account.uid == 0}>
+                                    {_("Terminate Session")}
+                                </Button>
+                                { "\n" }
+                                <Button isDisabled={account.uid == 0} variant="danger" id="account-delete"
+                                      onClick={() => delete_account_dialog(account)}>
+                                    {_("Delete")}
+                                </Button>
+                            </CardActions>
                             }
-                            <tr>
-                                <th scope="row"><label htmlFor="account-last-login">{_("Last Login")}</label></th>
-                                <td><output id="account-last-login">{last_login}</output></td>
-                            </tr>
-                            <tr>
-                                <th scope="row"><label htmlFor="account-locked">{_("Access")}</label></th>
-                                <td>
-                                    <div className="account-column-one">
-                                        <div className="checkbox" data-container="body">
-                                            <label>
-                                                <input type="checkbox" id="account-locked"
-                                   disabled={!superuser.allowed || edited_locked != null}
-                                   checked={edited_locked != null ? edited_locked : details.locked}
-                                   onChange={event => change_locked(event.target.checked)} />
-                                                <span>{_("Lock Account")}</span>
-                                            </label>
+                            <CardTitle id="account-title">{title_name}</CardTitle>
```

Comment:
The cart titles should probably have a heading inside.

Also, the actions should come after the titles, not relying on changing the order via CSS. (It's important to have document order for all types of a11y: keyboard navigation, screen reader, etc.)

Reply:
(Same should apply to the other cards.)